pkg/apis/cnsoperator/config/cnsregistervolume_crd.yaml

```diff
@@ -10,60 +10,59 @@ spec:
-          properties:
-            registered:
-              description: Indicates the volume is successfully imported.
-              type: boolean
-            error:
-              description: The last error encountered during import operation, if any.
-              type: string
-  version: v1alpha1
   versions:
   - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: CnsRegisterVolume is the Schema for the cnsregistervolumes API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            type: object
+            description: CnsRegisterVolumeSpec defines the desired state of CnsRegisterVolume
+            properties:
+              pvcName:
+                description: Name of the PVC
+                type: string
+                pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$'
+              volumeID:
+                description: VolumeID indicates an existing vsphere volume to be imported
+                  into Project Pacific cluster
+                type: string
+                pattern: '^[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}$'
+              accessMode:
+                description: AccessMode contains the actual access mode the volume backing
+                  the CnsRegisterVolume has
+                type: string
+              diskURLPath:
+                description: DiskUrlPath is URL path to an existing block volume to be
+                  imported into Project Pacific cluster.
+                type: string
+                pattern: '^(http[s]?:\/\/)?([^\/\s]+\/folder\/)(.*)$'
+            required:
+            - pvcName
+            type: object
+          status:
+            type: object
+            description: CnsRegisterVolumeStatus defines the observed state of CnsRegisterVolume
+            properties:
+              registered:
+                description: Indicates the volume is successfully imported.
+                type: boolean
+              error:
+                description: The last error encountered during import operation, if any.
+                type: string
```

Comment:
schema. openAPIV3Schema.type is a required field and must be set without which CRD will fail to create.
Please set `type: object` under openAPIV3Schema.